mlir_graphblas/src/test/GraphBLAS/lower_transpose_swap.mlir

```diff
@@ -0,0 +1,87 @@
+// RUN: graphblas-opt %s | graphblas-opt --graphblas-lower | FileCheck %s
+
+#CSR64 = #sparse_tensor.encoding<{
+  dimLevelType = [ "dense", "compressed" ],
+  dimOrdering = affine_map<(i,j) -> (i,j)>,
+  pointerBitWidth = 64,
+  indexBitWidth = 64
+}>
+ 
+
```

Comment:
Was this whole test manually written? 

If so, it seems like this sort of test would be hard to maintain. Maybe we should make it not necessarily check every single line and have a separate test verify that we get the same behavior once compiled to make sure all the lowering does what we want.

Reply:
This test was generated using the utility script in the MLIR source tree and the previously tested Jinja code from functions.py.  The goal of this test was to confirm that the C++ builder code produced something structurally identical to the hand-written MLIR we had before.  (This actually caught several transcription bugs.)  I agree this test is fragile, and maybe should be deleted eventually.